buildscripts/travis-build.sh

```diff
@@ -33,8 +37,21 @@ fi
 #make golint-travis
 #rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
 
+echo "Running : make format"
+make format
+rc=$?; if [[ $rc != 0 ]]; then echo "make format failed"; exit $rc; fi
+checkGitDiff "make format"
+printf "\n"
+
+echo "Running : make generated_files"
+make generated_files
+rc=$?; if [[ $rc != 0 ]]; then echo "make generated_files"; exit $rc; fi
+checkGitDiff "make generated_files"
+printf "\n"
+
 ./buildscripts/test-cov.sh
 rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
+checkGitDiff "make test"
```

Comment:
this change not required... make test will not be having any diff

Reply:
removed